ansible_collections/arista/avd/docs/release-notes/4.x.x.md

```diff
@@ -42,6 +42,8 @@ Pull request:
 - AVD version 5.0.0 will drop support for Python version 3.9. The decision has been taken to remove Python version 3.9 support in AVD collection to anticipate its removal in `ansible-core`.
   `ansible-core` version 2.15 End-Of-Life is scheduled for November 2024 and it will be the last `ansible-core` version supporting Python version 3.9 as documented here: https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#ansible-core-support-matrix.
 
+- In AVD version 5.0.0, the `eos_designs.jsonschema.json` and `eos_cli_config_gen.jsonschema.json` will not be generated anymore. These schemas are not being used and have never been complete.
```

Comment:
```suggestion
- In AVD version 5.0.0, the `eos_designs.jsonschema.json` and `eos_cli_config_gen.jsonschema.json` will no longer be generated. These schemas are not being used and have never been completed.
```

Reply:
This was added under 4.9.0, I'm thinking perhaps we need to move the `Deprecations` section at the top outside of a specific release with H2 heading?

Reply:
I would keep complete instead of completed